### PR TITLE
Leaderboard add Design

### PR DESCRIPTION
**Pull request by XRichardroidX**
In this project, I designed this project with CSS animation moving the time horizontally with changing colors.

### CODE REVIEW

File: src/assets/stylesheets/style.css [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+  font-size: 1.2rem;
+  border: 3px solid black;
+  box-shadow: 5px 5px;
+  border-radius: 7%;
+  cursor: pointer;
+}
+
+.btn-right {
+  float: right;
+}
```
Comment (rloterh) [inline src/assets/stylesheets/style.css:58]: - [ ] The submit button should be aligned to the right, together with the form elements as can be seen in the given [wireframe](https://github.com/microverseinc/curriculum-javascript/blob/main/leaderboard/images/leaderboard_wireframe.png). Kindly ensure to improve upon this aspect.


**Your design:**        |  **The Wireframe design:**
:-------------------------:|:-------------------------:
Your desgin |  Home Page

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Richard, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 🚦 🚦 🏃‍♂️   

## Highlights 👍  
- Awesome job with design implementation so far!

## Required Changes ♻️
_Kindly check the inline comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

**You can also consider:**
- Kindly ensure to add the _Future Features_ section to your ReadMe, which should include 1-3 potential features to be added.
- Kindly endeavor to remove the guide note below the MIT license.

Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by julie-ify — APPROVED** [review]
Hi @XRichardroidX,

## Highlights!
- Good feature branch name ✔️ 
- No linter errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 
- Good code organization and working app ✔️
- Descriptive README ✔️ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Status: Approved ✔️ 

Cheers and Happy coding!👏👏👏

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/assets/stylesheets/style.css (hunk 1 above)
Comment (XRichardroidX) [reply src/assets/stylesheets/style.css:58]: Thank you for the correction, Lol didn't notice it, I guess I have a mind for functionality

**Pull request merged**